ext/opcache/ZendAccelerator.c

```diff
@@ -3725,19 +3725,6 @@ static zend_result preload_resolve_deps(preload_error *error, const zend_class_e
 	return SUCCESS;
 }
 
-static zend_result preload_update_constant(zval *val, zend_class_entry *scope)
```

Comment:
Hm actually, if this code is dropped, can't we still run into an issue with the `ZEND_AST_NEW` case, which also creates objects?

Reply:
Oh, right, I forgot about that. We'll need the same check there then (as this still doesn't cover `const FOO = [new Foo()];`).

Reply:
Actually, `new` is not allowed in constants, property default values or static property default values. So I think this should be fine.

Reply:
True...